routes/comment.js

```diff
@@ -2,7 +2,8 @@ import express from 'express';
 import CommentValidation from '../middlewares/CommentValidation';
 import ensureAuthenticated from '../middlewares/checkAuthentication';
```

Comment:
`ensureAuthentication` does not match the file name which is `checkAuthentication`